### ISSUE TITLE
Installation failure: `PIP.md` is missing

### ISSUE THREAD

**Issue by vladimyr** [KuroLabs/Airshare#15]
Installation fails due to missing `PIP.md` markdown file referenced inside [`setup.py`](https://github.com/KuroLabs/Airshare/blob/0.1.4/setup.py#L3-L4):
```py
with open("PIP.md", "r") as f:
    long_description = f.read()
```

I went to [pypi.org](https://pypi.org/project/Airshare/#files) and downloaded the latest [source](https://files.pythonhosted.org/packages/a2/b3/2c8a666dbe4ac16447fd657f1f345d5833d890c96e94e6accfa32932a793/Airshare-0.1.4.tar.gz) to verify its contents:
```
➜ curl -s https://files.pythonhosted.org/packages/a2/b3/2c8a666dbe4ac16447fd657f1f345d5833d890c96e94e6accfa32932a793/Airshare-0.1.4.tar.gz | tar tzf - | awk '{print $NF}'
Airshare-0.1.4/
Airshare-0.1.4/Airshare.egg-info/
Airshare-0.1.4/Airshare.egg-info/PKG-INFO
Airshare-0.1.4/Airshare.egg-info/SOURCES.txt
Airshare-0.1.4/Airshare.egg-info/dependency_links.txt
Airshare-0.1.4/Airshare.egg-info/entry_points.txt
Airshare-0.1.4/Airshare.egg-info/requires.txt
Airshare-0.1.4/Airshare.egg-info/top_level.txt
Airshare-0.1.4/MANIFEST.in
Airshare-0.1.4/PKG-INFO
Airshare-0.1.4/README.md
Airshare-0.1.4/airshare/
Airshare-0.1.4/airshare/__init__.py
Airshare-0.1.4/airshare/cli.py
Airshare-0.1.4/airshare/exception.py
Airshare-0.1.4/airshare/qrcode.py
Airshare-0.1.4/airshare/receiver.py
Airshare-0.1.4/airshare/sender.py
Airshare-0.1.4/airshare/static/
Airshare-0.1.4/airshare/static/download.html
Airshare-0.1.4/airshare/static/text.html
Airshare-0.1.4/airshare/static/upload.html
Airshare-0.1.4/airshare/utils.py
Airshare-0.1.4/setup.cfg
Airshare-0.1.4/setup.py
```
As you can see `PIP.md` is missing. Please publish a patch to PyPI with all required source files or at least ensure everything is there for upcoming versions.

**Comment by mohanpierce99** [comment]
Somethings wrong just tested the pip install in multiple machines and every one of em works fine. I'm not sure what's the issue, Not able to reproduce the issue but let me get onto this.

**Comment by vladimyr** [comment]
You probably won't be able to reproduce it with regular `pip` install.
I experienced it during the creation of Homebrew formula for Airshare. Homebrew installs python packages inside dedicated virtual environments. Airshare is installed using given `pip` install command:

<details>
<summary>
Click to see output<br>
<code>/usr/local/Cellar/airshare/0.1.4/libexec/bin/pip install -v --no-deps --no-binary :all: --ignore-installed /private/tmp/airshare-20200604-45643-9dl1za/Airshare-0.1.4</code></summary>

```
Non-user install because user site-packages disabled
Created temporary directory: /private/tmp/pip-ephem-wheel-cache-t18_f1st
Created temporary directory: /private/tmp/pip-req-tracker-dencal4q
Initialized build tracking at /private/tmp/pip-req-tracker-dencal4q
Created build tracker: /private/tmp/pip-req-tracker-dencal4q
Entered build tracker: /private/tmp/pip-req-tracker-dencal4q
Created temporary directory: /private/tmp/pip-install-gdtvjear
Processing /private/tmp/airshare-20200604-45643-9dl1za/Airshare-0.1.4
  Created temporary directory: /private/tmp/pip-req-build-ax3govp0
  Added file:///private/tmp/airshare-20200604-45643-9dl1za/Airshare-0.1.4 to build tracker '/private/tmp/pip-req-tracker-dencal4q'
    Running setup.py (path:/private/tmp/pip-req-build-ax3govp0/setup.py) egg_info for package from file:///private/tmp/airshare-20200604-45643-9dl1za/Airshare-0.1.4
    Created temporary directory: /private/tmp/pip-pip-egg-info-vmll32gh
    Running command python setup.py egg_info
    running egg_info
    creating /private/tmp/pip-pip-egg-info-vmll32gh/Airshare.egg-info
    writing /private/tmp/pip-pip-egg-info-vmll32gh/Airshare.egg-info/PKG-INFO
    writing dependency_links to /private/tmp/pip-pip-egg-info-vmll32gh/Airshare.egg-info/dependency_links.txt
    writing entry points to /private/tmp/pip-pip-egg-info-vmll32gh/Airshare.egg-info/entry_points.txt
    writing requirements to /private/tmp/pip-pip-egg-info-vmll32gh/Airshare.egg-info/requires.txt
    writing top-level names to /private/tmp/pip-pip-egg-info-vmll32gh/Airshare.egg-info/top_level.txt
    writing manifest file '/private/tmp/pip-pip-egg-info-vmll32gh/Airshare.egg-info/SOURCES.txt'
    reading manifest file '/private/tmp/pip-pip-egg-info-vmll32gh/Airshare.egg-info/SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    writing manifest file '/private/tmp/pip-pip-egg-info-vmll32gh/Airshare.egg-info/SOURCES.txt'
  Source in /private/tmp/pip-req-build-ax3govp0 has version 0.1.4, which satisfies requirement Airshare==0.1.4 from file:///private/tmp/airshare-20200604-45643-9dl1za/Airshare-0.1.4
  Removed Airshare==0.1.4 from file:///private/tmp/airshare-20200604-45643-9dl1za/Airshare-0.1.4 from build tracker '/private/tmp/pip-req-tracker-dencal4q'
Skipping wheel build for Airshare, due to binaries being disabled for it.
Installing collected packages: Airshare
  Created temporary directory: /private/tmp/pip-record-nx30ojik
    Running setup.py install for Airshare: started
    Running command /usr/local/Cellar/airshare/0.1.4/libexec/bin/python3.8 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/tmp/pip-req-build-ax3govp0/setup.py'"'"'; __file__='"'"'/private/tmp/pip-req-build-ax3govp0/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/tmp/pip-record-nx30ojik/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/Cellar/airshare/0.1.4/libexec/bin/../include/site/python3.8/Airshare
    running install
    running build
    running build_py
    creating build
    creating build/lib
    creating build/lib/airshare
    copying airshare/sender.py -> build/lib/airshare
    copying airshare/exception.py -> build/lib/airshare
    copying airshare/receiver.py -> build/lib/airshare
    copying airshare/__init__.py -> build/lib/airshare
    copying airshare/qrcode.py -> build/lib/airshare
    copying airshare/cli.py -> build/lib/airshare
    copying airshare/utils.py -> build/lib/airshare
    running egg_info
    writing Airshare.egg-info/PKG-INFO
    writing dependency_links to Airshare.egg-info/dependency_links.txt
    writing entry points to Airshare.egg-info/entry_points.txt
    writing requirements to Airshare.egg-info/requires.txt
    writing top-level names to Airshare.egg-info/top_level.txt
    reading manifest file 'Airshare.egg-info/SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    writing manifest file 'Airshare.egg-info/SOURCES.txt'
    creating build/lib/airshare/static
    copying airshare/static/download.html -> build/lib/airshare/static
    copying airshare/static/text.html -> build/lib/airshare/static
    copying airshare/static/upload.html -> build/lib/airshare/static
    running install_lib
    creating /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare
    copying build/lib/airshare/sender.py -> /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare
    copying build/lib/airshare/exception.py -> /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare
    copying build/lib/airshare/receiver.py -> /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare
    copying build/lib/airshare/__init__.py -> /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare
    copying build/lib/airshare/qrcode.py -> /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare
    copying build/lib/airshare/cli.py -> /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare
    copying build/lib/airshare/utils.py -> /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare
    creating /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare/static
    copying build/lib/airshare/static/text.html -> /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare/static
    copying build/lib/airshare/static/download.html -> /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare/static
    copying build/lib/airshare/static/upload.html -> /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare/static
    byte-compiling /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare/sender.py to sender.cpython-38.pyc
    byte-compiling /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare/exception.py to exception.cpython-38.pyc
    byte-compiling /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare/receiver.py to receiver.cpython-38.pyc
    byte-compiling /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare/__init__.py to __init__.cpython-38.pyc
    byte-compiling /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare/qrcode.py to qrcode.cpython-38.pyc
    byte-compiling /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare/cli.py to cli.cpython-38.pyc
    byte-compiling /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/airshare/utils.py to utils.cpython-38.pyc
    running install_egg_info
    Copying Airshare.egg-info to /usr/local/Cellar/airshare/0.1.4/libexec/lib/python3.8/site-packages/Airshare-0.1.4-py3.8.egg-info
    running install_scripts
    Installing airshare script to /usr/local/Cellar/airshare/0.1.4/libexec/bin
    writing list of installed files to '/private/tmp/pip-record-nx30ojik/install-record.txt'
    Running setup.py install for Airshare: finished with status 'done'
Successfully installed Airshare-0.1.4
Removed build tracker: '/private/tmp/pip-req-tracker-dencal4q'
```
</details>

It definitely runs `setup.py` under the hood and it fails because there is no `PIP.md`.

**Comment by vladimyr** [comment]
BTW log that I just posted ends up with:
```
    Running setup.py install for Airshare: finished with status 'done'
```
but that's just due to fact I created empty `PIP.md` as a hacky workaround.

**Comment by nandahkrishna** [comment]
This is just _slightly_ related to this issue, but the Homebrew/homebrew-core notability audit might fail because we have < 30 forks 😅

Edit: I checked your PR in homebrew-core, @vladimyr. I'm kinda surprised the notability audit didn't fail (I think it's 100 stars, 30 watchers, 30 forks - maybe it was OR and not AND). Thanks for the PR, I'll fix the issue and let you know.

Meanwhile, I'll just have to add PIP.md to the MANIFEST file. That should prevent the error, I think.

**Comment by nandahkrishna** [comment]
I've released `0.1.5` on PyPI, and I've made sure that PIP.md is included. Could you update the homebrew-core Formula @vladimyr?

**Comment by vladimyr** [comment]
> I've released `0.1.5` on PyPI, and I've made sure that PIP.md is included. Could you update the homebrew-core Formula @vladimyr?

Thanks! Sure, I'll do it tonight.

**Comment by vladimyr** [comment]
> Edit: I checked your PR in homebrew-core, @vladimyr. I'm kinda surprised the notability audit didn't fail (I think it's 100 stars, 30 watchers, 30 forks - maybe it was OR and not AND). Thanks for the PR, I'll fix the issue and let you know.

I believe you are referring to https://github.com/Homebrew/brew/blob/49324ad/Library/Homebrew/utils/notability.rb#L18-L21 If any of those thresholds is passed it is declared notable enough, so it is OR. 😉 

> I've released `0.1.5` on PyPI, and I've made sure that PIP.md is included. Could you update the homebrew-core Formula @vladimyr?

Done ✅ 

The issue has been resolved, thus closing it 👍

**Comment by nandahkrishna** [comment]
Amazing, thanks!
